src/transaction/boot_sr.c

```diff
@@ -2264,6 +2265,13 @@ boot_restart_server (THREAD_ENTRY * thread_p, bool print_restart, const char *db
 
   event_log_init (db_name);
 
+#if defined(SERVER_MODE)
+  if (mmon_initialize (db_name) != NO_ERROR)
+    {
+      goto error;
```

Comment:
Don't you need to add `mmon_finalize ();` after error: label?

Reply:
There is no need to add the function in error handling because `mmon_finalize()` is called after the function (boot_restart_server()) exits.